cmake/add_project_smqtk.cmake

```diff
@@ -24,15 +21,14 @@ else()
   set( PYTHON_BASEPATH
     ${VIAME_BUILD_INSTALL_PREFIX}/lib/python${PYTHON_VERSION}${PYTHON_ABIFLAGS} )
   set( CUSTOM_PYTHONPATH
-    ${PYTHON_BASEPATH}/site-packages:${PYTHON_BASEPATH}/dist-packages:$ENV{PYTHONPATH} )
+    ${PYTHON_BASEPATH}/site-packages:${PYTHON_BASEPATH}/dist-packages )
   set( CUSTOM_PATH
-    ${VIAME_BUILD_INSTALL_PREFIX}/bin:$ENV{PATH} )
+    ${VIAME_BUILD_INSTALL_PREFIX}/bin )
   set( SMQTK_PYTHON_INSTALL
     ${CMAKE_COMMAND} -E env PYTHONPATH=${CUSTOM_PYTHONPATH}
-                        env PATH=${CUSTOM_PATH}
-                        env PYTHONUSERBASE=${VIAME_BUILD_INSTALL_PREFIX}
+                        PATH=${CUSTOM_PATH}
+                        PYTHONUSERBASE=${VIAME_BUILD_INSTALL_PREFIX}
       ${PYTHON_EXECUTABLE} -m ${SMQTK_PIP_CMD} )
-endif()
 
```

Comment:
Why do you still have the first 2 spaces if the if statement was removed?

Reply:
In the old version before his latest commit he had an extra indentation for the entire if block but was fixed and merged